pyodide_build/tests/test_config.py

```diff
@@ -43,6 +47,8 @@ def test_load_config_from_file(self, tmp_path, reset_env_vars, reset_cache):
                                   skip_emscripten_version_check = "1"
                                   xbuildenv_path = "my_custom/xbuildenv_path"
                                   ignored_build_requirements = "cmake foo bar"
+                                  "interpreter" = "$(PYODIDE_INTERPRETER)"
+                                  "package_index" = "$(PYODIDE_PACKAGE_INDEX)"
```

Comment:
I don't think we expect people to overwrite these configs.

Reply:
Yes, this is with the in-tree case, I'll drop it. What should be the scenario with the out-of-tree case?

Reply:
I don't think people would want to overwrite these variables in out-of-tree as well. The interpreter is coming from the xbuildenv, and the package_index is created by `pyodide venv`. I guess these variables are used mostly for getting the path for these values rather than overwriting it?